projects/fastify-capture/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2020",
```

Comment:
is there a specific reason why not `ES2021`?

Reply:
Oops - fixing